Exiled.Events/Handlers/Map.cs

```diff
@@ -7,14 +7,15 @@
 
 namespace Exiled.Events.Handlers
 {
-    using Exiled.API.Extensions;
     using Exiled.Events.EventArgs;
+    using Exiled.Events.Extensions;
+
```

Comment:
Inconsistent spacing. Some files have a space blank line before static using declarations, some do not. I don't care either way or the other but make them all the same :pepeangry:

Reply:
Yes, this happened when changing from `API` to `Events`, I'll fix it.